packages/manager/apps/pci/package.json

```diff
@@ -45,12 +45,12 @@
     "ovh-api-services": "^7.0.0",
     "ovh-common-style": "^5.0.0",
     "ovh-manager-webfont": "^1.2.0",
-    "ovh-ui-angular": "^3.3.3",
+    "ovh-ui-angular": "^3.5.1",
```

Comment:
Should be upgraded in all `apps` 
`yarn upgrade-interactive --latest ovh-ui-angular`